components/CallToActionSection.vue

```diff
@@ -64,7 +105,22 @@ export default {
       type: Object,
       default: () => ({}),
     },
+    ctaValue: {
+      type: Object,
+      default: () => ({})
+    }
   },
+  computed: {
+    title(){
+      return this.ctaValue.title || []
+    }, 
```

Comment:
How come title is array? If it array shouldn't the name be `titles`?

Reply:
Nope, it's a single title.
<img width="1076" alt="Screenshot 2024-07-04 at 11 25 21 AM" src="https://github.com/formester/formester-site/assets/79869032/08aac2cf-bb22-4f56-b397-52a9534e9d87">
It's an array because we want to know whether the particular part of the title should be highlighted or not.